internal/golden/golden.go

```diff
@@ -350,7 +365,10 @@ func addEmptyMarker(p string) error {
 			if err != nil {
 				return err
 			}
-			f.Close()
+			err = f.Close()
+			if err != nil {
+				return err
+			}
```

Comment:
:eyes:  (you already know my comment...)

Reply:
fixed